junwoo/controllers/postController.js

```diff
@@ -0,0 +1,52 @@
+//controller/postController.js
+
+const postService = require("../services/postService");
+
+const postUp = async (req, res) => {
+  try {
+    const { title, content, image, user_id } = req.body;
```

Comment:
- `user_id`를 camelCase로 받을 수 있고 사용하도록 수정해주세요.
```suggestion
    const { title, content, image, userId } = req.body;
```